workspaces/ui-v2/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM library/nginx:alpine
+ARG APP
```

Comment:
if we're only building one project from this code for now you could hardcode the value build directory on the next line. the intention here was originally to disambiguate the various builds produced from the UI code, but still allow apidocs and demo to share the docker image bits.

Reply:
I do think that eventually we'll want multiple builds here: demo is eventually going to come from this, for example. So I'd rather leave it parameterized I think